app/webpack/observations/show/components/observation_field_value.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { PropTypes } from "react";
 import { Popover, OverlayTrigger } from "react-bootstrap";
 import SplitTaxon from "../../../shared/components/split_taxon";
+import ObservationFieldText from "../../../shared/components/observation_field_text";
```

Comment:
This new component is probably unnecessary. https://github.com/inaturalist/inaturalist/blob/master/app/webpack/shared/components/user_text.jsx might do the trick.